src/renderer/apps/apps.ts

```diff
@@ -23,6 +24,7 @@ export const AppList: AppDefinition[] = [
 	// Optional Apps.
 	Mento,
 	Celovote,
+	MultiSend,
```

Comment:
This isn't sorted alphabetically, it is sorted more or less by date of addition. Can you just add this at the end of the list.